aag/weather.py

```diff
@@ -748,9 +748,11 @@ def calculate_and_set_pwm(self):
 
                 self.set_pwm(new_pwm)
 
-    def make_safety_decision(self, current_values):
+    def make_safety_decision(self, current_values, ignore=False):
```

Comment:
It's a bit better to set this to `None` rather than, which indicates we don't have a default but is (somewhat) consistent with the same data type (list of strings) as opposed to `False`.

```suggestion
    def make_safety_decision(self, current_values, ignore=None):
```